### ISSUE TITLE
ICD-9 571.5 does not map to a standard concept

### ISSUE THREAD

**Issue by schuemie** [OHDSI/Vocabulary-v5.0#77]
Why does 571.5 (Cirrhosis of liver without mention of alcohol) not map to a standard concept? (e.g. Cirrhosis of liver (4064161) )

**Comment by ericaVoss** [comment]
**20160620**

This is still an issue:

```{sql}
SELECT *
FROM CONCEPT c
	JOIN CONCEPT_RELATIONSHIP cr
		ON concept_id_1 = c.concept_id
	JOIN CONCEPT c2
		ON c2.concept_id = cr.concept_id_2
		AND c2.STANDARD_CONCEPT = 'S'
WHERE c.CONCEPT_CODE = '571.5'
```

Returns 0 rows.

**Comment by cgreich** [comment]
I know. On the backlog.

**Comment by ericaVoss** [comment]
Also discussed in Issue #88.  This is still a problem in 20160817.

**Comment by ericaVoss** [comment]
In 20161107.

**Comment by ericaVoss** [comment]
In 20161218

**Comment by aostropolets** [comment]
@schuemie @ericaVoss 
maps to 194692 Cirrhosis - non-alcoholic
...trying to close all those issues :)

**Comment by ericaVoss** [comment]
Fixed in 20170920 but I cannot close it.  @schuemie can you close it please.

**Comment by ericaVoss** [comment]
Still fixed in 20171201, @schuemie  could you please close this issue.
